components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/internal/http/api/ConfigurationLoader.java

```diff
@@ -79,10 +105,27 @@
                 if (handlerElem.getAttribute(CLASS_Q) != null) {
                     String className = handlerElem.getAttributeValue(CLASS_Q);
                     if (!className.isEmpty()) {
-                        InternalAPI internalApi = createAPI(className);
+                        InternalAPI internalApi = createApi(className);
                         if (internalApi != null) {
```

Comment:
Can createApi return null? It wil either return a valid API or throw and exception